src/AgentVisibilityContext.tsx

```diff
@@ -83,9 +83,7 @@ export const VisibilityProvider = ({
     setHiddenState(newHiddenState);
   };
 
-  const hideOrRevealAllAgents = (
-    hiddenState: HiddenOrHighlightedState
-  ): void => {
+  const setAllAgentVisibility = (hiddenState: ActiveState): void => {
```

Comment:
Consider adding a method comment here or in the type definition on line 23 to explain the expected behavior for `hiddenState` (`hiddenState === ActiveState.ACTIVE`) if agents should be hidden.

Reply:
(If you do it as a `/**/` comment it'll show up w/ Intellisense too!)